### PR TITLE
#1387: "Uncaught TypeError: Cannot read property 'complete' of undefined" appears in dev console if save Previewed image as a new View and open this View on another page

### LINKED ISSUE #1387: "Uncaught TypeError: Cannot read property 'complete' of undefined" appears in dev console if save Previewed image as a new View and open this View on another page
### Preconditions (*)
1. Install Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset

### Steps to reproduce (*)
1. Go to **Content - Media Gallery**
2. Click **Search Adobe Stock**
3. Go to the first page from the grid, and select any image to open its Preview
4. Click on **Default View** (bookmarks view)
5. Click **Save View As...**, 
6. Click **Save all Changes** :arrow_right: 
7. Switch to the _Default View_
8. Go to the second page in Adobe Stock grid
9. Switch to the previously created _New View_

### Expected result (*)
No error message in dev console

### Actual result (*)
:x: _Uncaught TypeError: Cannot read property 'complete' of undefined_ appears in dev console 
uncought_complete_view_2520

### PR DESCRIPTION
**Pull request by joweecaquicla**
### Description (*)

This PR will introduce the fix to prevent the error displaying in the console. 
### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/adobe-stock-integration/issues/1387: "Uncaught TypeError: Cannot read property 'complete' of undefined" appears in dev console if save Previewed image as a new View and open this View on another page

### Manual testing scenarios (*)

1. Go to Content - Media Gallery
2. Click Search Adobe Stock
3. Go to the first page from the grid, and select any image to open its Preview
4. Click on Default View (bookmarks view)
5. Click Save View As...,
6. Click Save all Changes ➡️
7. Switch to the Default View
8. Go to the second page in Adobe Stock grid
9. Switch to the previously created New View

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by joweecaquicla** [comment]
@magento run all tests

File: app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js [hunk 1]
```diff
@@ -141,19 +156,42 @@ define([
...
+                    }.bind(this));
+                }
             }
+        },
 
-            this.lastOpenedImage(record._rowIndex);
+        /**
+         * Update displayed record
+         *
+         * @param items
```
Comment (sivaschenko) [inline app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js]: I believe missing paramter type will result in the static test failure

File: app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js [hunk 2]
```diff
@@ -141,19 +156,42 @@ define([
...
+                        this.updateHeight();
+                        this.scrollToPreview();
+                    }.bind(this));
+                }
             }
+        },
 
-            this.lastOpenedImage(record._rowIndex);
+        /**
+         * Update displayed record
```
Comment (sivaschenko) [inline app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js]: It's better to make comments at least a bit more descriptive than method name if possible
```suggestion
         * Update preview displayed record data from the new items data if the preview is expanded
```

**Comment by joweecaquicla** [comment]
@magento run all tests

**Comment by joweecaquicla** [comment]
@magento run all tests

**Comment by joweecaquicla** [comment]
@magento run all tests

**Comment by joweecaquicla** [comment]
@magento run all tests

**Comment by joweecaquicla** [comment]
@magento run Functional Tests EE, Functional Tests B2B

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-8065](https://jira.corp.magento.com/browse/ENGCOM-8065) has been created to process this Pull Request
:eight_spoked_asterisk: @sivaschenko, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by chalov-anton** [comment]
:heavy_check_mark: QA Passed

The View is switched correctly: 
- switched to the correct page
- image Preview is displayed

Manual testing scenario https://studio.cucumber.io/projects/131313/test-plan/folders/1337102/scenarios/5126336

**Pull request merged**